src/libtorch.cc

```diff
@@ -88,11 +89,14 @@ class ModelState : public BackendModel {
   // Parses and validates parameters in config
   TRITONSERVER_Error* ParseParameters();
 
-  // Flag to indicate whether optimized execution is disabled
+  // Flag to indicate whether optimized execution is disabled. Defaults to false.
   bool disable_optimized_execution_;
 
-  // Flag to indicate whether inference mode is enabled
+  // Flag to indicate whether inference mode is enabled. Defaults to false.
   bool inference_mode_;
```

Comment:
Having enables seems like it would be more typical than having disables. But should be consistent across all 3. And name them all consistently as enable_* or disable_*

Reply:
Since the default value for these 3 flags are different do we still want to name them consistently? 
optimized_execution is on by default.
inference_mode is off by default.
nvfuser is off by default.

Reply:
More important to have consistent naming then to match the defaults.

Reply:
Fixed